cli/dcoscli/metrics.py

```diff
@@ -125,7 +127,7 @@ def _node_summary_data(datapoints):
     disk_free = _get_datapoint_value(
         datapoints, 'filesystem.capacity.used', {'path': '/'})
```

Comment:
the variable is called disk_free while it seems to be disk_used (`filesystem.capacity.used`).

Reply:
Fixed :)